### PR TITLE
Add prettier step to sbwf/module guidelines and update all examples to 2.8

### PR DESCRIPTION
**Pull request by jfy133**

### CODE REVIEW

**Comment by jfy133** [comment]
@nf-core-bot fix linting

File: markdown/developers/modules.md [hunk 1]
```diff
@@ -264,50 +268,57 @@ Please follow the steps below to run the tests locally:
...
+   ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 98%]
+   tests/modules/fastqc/test.yml ........
+   Keeping temporary directories and logs. Use '--kwd' or '--keep-workflow-wd' to disable this behaviour.
+   ============================= 10 passed, 751 skipped, 479 warnings in 50.76s =============================
+
+   ```
+
    - See [docs on running pytest-workflow](https://pytest-workflow.readthedocs.io/en/stable/#running-pytest-workflow) for more info.
+   - If the module is also used in subworkflows, all subworkflow tests containing the module will also be executed.
+   ```
```
Comment (mirpedrol) [inline markdown/developers/modules.md:321]:
```suggestion
```

File: markdown/developers/modules.md [hunk 2]
```diff
@@ -264,50 +268,57 @@ Please follow the steps below to run the tests locally:
...
-     ============================= 10 passed, 751 skipped, 479 warnings in 50.76s =============================
+
+                                              ,--./,-.
+              ___     __   __   __   ___     /,-._.--~\
+        |\ | |__  __ /  ` /  \ |__) |__         }  {
+        | \| |       \__, \__/ |  \ |___     \`-._,-`-,
+                                              `._,._,'
+
+        nf-core/tools version 2.8 - https://nf-co.re
      ```
```
Comment (mirpedrol) [inline markdown/developers/modules.md:279]:
```suggestion
```

File: markdown/developers/subworkflows.md [hunk 3]
```diff
@@ -63,24 +63,23 @@ We have implemented a number of commands in the `nf-core/tools` package to make
...
+                                             `._,._,'
 
- nf-core/tools version 2.7.dev0 - https://nf-co.re
+       nf-core/tools version 2.8 - https://nf-co.re
 
 
- INFO     Repository type: modules
-```
+   INFO     Repository type: modules
+   INFO     Press enter to use default values (shown in brackets) or type your own responses. ctrl+click underlined text to open links.
```
Comment (mirpedrol) [inline markdown/developers/subworkflows.md:81]:
```suggestion
   INFO     Press enter to use default values (shown in brackets) or type your own responses. ctrl+click underlined text to open links.
   INFO     Created / edited following files:                                                                                 
              ./subworkflows/nf-core/bam_sort_stats_samtools/main.nf                                                                            
              ./subworkflows/nf-core/bam_sort_stats_samtools/meta.yml                                                                           
              ./tests/subworkflows/nf-core/bam_sort_stats_samtools/main.nf                                                                      
              ./tests/subworkflows/nf-core/bam_sort_stats_samtools/test.yml                                                                     
              ./tests/subworkflows/nf-core/bam_sort_stats_samtools/nextflow.config                                                              
              ./tests/config/pytest_modules.yml   
```

File: markdown/developers/subworkflows.md [hunk 4]
```diff
@@ -216,24 +218,28 @@ Please follow the steps below to run the tests locally:
...
-   nf-core/tools version 2.7.dev0 - https://nf-co.re
+    nf-core/tools version 2.8 - https://nf-co.re
 
 
-   INFO     Repository type: modules
+    INFO     Press enter to use default values (shown in brackets) or type your own responses
+    ? Choose software profile (Use arrow keys)
+    » Docker
+      Singularity
+      Conda
```
Comment (mirpedrol) [inline markdown/developers/subworkflows.md:242]:
```suggestion
    ? Choose software profile Docker
    INFO     Setting environment variable '$PROFILE' to 'conda'
    NFO     Running pytest for subworkflow 'bam_sort_stats_samtools'
```

File: markdown/developers/subworkflows.md [hunk 5]
```diff
@@ -117,28 +116,31 @@ All of the files required to add the subworkflow to `nf-core/modules` will be cr
...
 
-   nf-core/tools version 2.7.dev0 - https://nf-co.re
+       nf-core/tools version 2.8 - https://nf-co.re
 
 
-   INFO     Repository type: modules
+   INFO     Press enter to use default values (shown in brackets) or type your own
+          responses
+   Test YAML output path (- for stdout)
+
```
Comment (mirpedrol) [inline markdown/developers/subworkflows.md:136]:
```suggestion
   INFO     Press enter to use default values (shown in brackets) or type your own 
      responses
   Test YAML output path (- for stdout) (tests/subworkflows/nf-core/bam_sort_stats_samtools/test.yml):
   INFO     Looking for test workflow entry points: 'tests/subworkflows/nf-core/bam_sort_stats_samtools/main.nf'
   INFO     Building test meta for entry point 'test_bam_sort_stats_samtools_single_end'
   Test name (bam_sort_stats_samtools test_bam_sort_stats_samtools_single_end):
   Test command (nextflow run ./tests/subworkflows/nf-core/bam_sort_stats_samtools -entry 
      test_bam_sort_stats_samtools_single_end -c ./tests/config/nextflow.config):
   Test tags (comma separated):
   Test output folder with results (leave blank to run test):
   ? Choose software profile Docker
   INFO     Setting env var '$PROFILE' to 'docker'
   INFO     Running 'bam_sort_stats_samtools' test with command:                                                              
         nextflow run ./tests/subworkflows/nf-core/bam_sort_stats_samtools -entry       
            test_bam_sort_stats_samtools_single_end  
            -c ./tests/config/nextflow.config --outdir /var/folders/lt/b3cs9y610fg_13q14dckwcvm0000gn/T/tmping28ow_ 
            -work-dir /var/folders/lt/b3cs9y610fg_13q14dckwcvm0000gn/T/tmportf0uab
```

**Review by mirpedrol — APPROVED** [review]
LGTM :) 
Just added a couple of suggestions regarding example outputs

**Review by scorreard — COMMENTED** [review]
Just added the prettier command for people who have it installed already or ppl using gitpod

**Comment by jfy133** [comment]
@nf-core-bot fix linting

**Review by scorreard — APPROVED** [review]
Looks great!! It will be super helpful, thanks!

**Pull request merged**